compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -1027,11 +1034,21 @@ OMR::X86::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode, TR::D
       case TR::vstore:
       case TR::vstorei:
       case TR::vsplats:
-      case TR::getvelem:
          if (dt == TR::Int32 || dt == TR::Int64 || dt == TR::Float || dt == TR::Double)
             return true;
          else
             return false;
+      /*
+       * GRA does not work with vector registers on 32 bit due to a bug where xmm registers are not being assigned
+       * This can potentially cause a performance problem in autosimd reductions
+       * This function is where AutoSIMD checks to see if getvelem is suppored for use in reductions
+       * getvelem is temporarily marked as unsupported until the GRA problem has been fixed.
```

Comment:
* Please use periods at the end of your sentences.

* Rather than say this fix is temporary, please reference the issue you created to track the resolution of the problem in the comment instead.  Part of the work in that issue should be to re-enable this code and remove the comment.